e2e-tests/tox.ini

```diff
@@ -1,37 +1,21 @@
 [tox]
 skipsdist = true
-envlist = py27, flake8
+envlist = py27
```

Comment:
Removing flake8 from this Tox env here breaks our Jenkins build, due to https://github.com/mozilla-services/socorro/blob/87eb5aa4d85e355739b1db13df729e2675ceba99/e2e-tests/Jenkinsfile#L29 - that 'Lint' stage should be removed.

Reply:
r+ once the Tox/flake8/Jenkinsfile issue has been addressed